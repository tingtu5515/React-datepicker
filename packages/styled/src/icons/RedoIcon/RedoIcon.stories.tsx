import React from 'react'
import {storiesOf} from '@storybook/react'
import {text, color} from '@storybook/addon-knobs'
import RedoIcon from '.'

storiesOf('Components/Icons', module).add('RedoIcon', () => (
  <RedoIcon
    color={color('Color', '#000')}
    height={text('Height', '30px')}
    width={text('Width', '30px')}
  />
))
