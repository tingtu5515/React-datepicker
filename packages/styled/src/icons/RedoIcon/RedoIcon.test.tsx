import * as React from 'react'
import {render} from 'react-testing-library'
import RedoIcon from '.'

test('should match snapshot', () => {
  const {container} = render(<RedoIcon height="30px" width="30px" color="red" />)
  expect(container).toMatchSnapshot()
})
