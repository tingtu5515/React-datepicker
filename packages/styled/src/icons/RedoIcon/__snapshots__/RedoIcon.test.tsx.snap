// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div>
  <svg
    class=""
    color="red"
    height="30px"
    viewBox="0 0 14 14"
    width="30px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M9.015 11.15c-.027-.18-.04-.39-.067-.585a3.958 3.958 0 0 1-4.48-.056C2.663 9.241 2.142 6.663 3.292 4.74c1.217-2.02 3.797-2.592 5.696-1.282.589.404 1.03.934 1.35 1.533l-1.216.808L13 7.917l-.174-4.556-1.056.696a5.812 5.812 0 0 0-1.846-2.062C7.25.155 3.64.935 1.901 3.765c-1.672 2.717-.95 6.382 1.605 8.194a5.535 5.535 0 0 0 5.616.501c0-.083 0-.167-.013-.264a9.193 9.193 0 0 0-.094-1.046z"
      fill="currentColor"
      fill-rule="nonzero"
    />
  </svg>
</div>
`;
